virtool/groups/data.py

```diff
@@ -63,12 +63,12 @@ async def create(self, group_id: str) -> Group:
 
         return Group(**base_processor(document), users=[])
 
-    async def update(self, group_id: str, permissions_update: Dict[str, str]) -> Group:
+    async def update(self, group_id: str, data: Dict[str, Union[str, dict]]) -> VTGroup:
```

Comment:
I want the data layer methods to take the Pydantic objects. That way we have typing on the `data` parameter.

Reply:
So permissions update should not be passed in? Instead it should be data: `EditGroupsSchema`?

Reply:
`data: EditGroupsSchema`

Yes, that's it.